kwave/utils/angular_spectrum.py

```diff
@@ -11,18 +14,23 @@
 from kwave.utils.tictoc import TicToc
 
 
+@beartype
 def angular_spectrum(
-        input_plane, dx, dt, z_pos, medium,
-        angular_restriction=True,
-        grid_expansion=0,
-        fft_length='auto',
-        data_cast='off',
-        data_recast=False,
-        reverse_proj=False,
-        absorbing=False,
-        plot_updates=False,
-        loops_for_time_est=5,
-        record_time_series=False
+        input_plane: NDArray[Shape["Dim1, Dim2, Dim3"], Float], 
```

Comment:
I'd prefer having expressive names. I do not always know what each dimension corresponds and did not want to use `x, y, z` everywhere because it ordering might be wrong. That is why I went with generic names. Can you share examples for the verbose names?

Reply:
"depth", "width", "height" are not much better than "x", "y", "z" but  might be more expressive.

Reply:
Are we sure that it is always in the same order ("depth", "width", "height")? If not, I'd propose we leave the code as is and revisit in the future, just because it will be large work to inspect the order everytime in this PR :)

Reply:
I am not sure so let's revisit it in the future. :)